src/Race.js

```diff
@@ -0,0 +1,59 @@
+import { Random, Console } from "@woowacourse/mission-utils";
+
+class Race {
+  #cars = [];
+  #winners = [];
+  #raceCount;
+
+
+  addRacingCar(car) {
+    this.#cars.push(car);
+  }
+
+  setRaceCount(raceCount) {
+    this.#raceCount = raceCount;
+  }
+
+  getRaceCount() {
+    return this.#raceCount;
+  }
+
+  racing() {
+    this.#cars.forEach(car => {
+      const random = Random.pickNumberInRange(0, 9);
+      // 4 이상일 경우 전진
+      car.forward(random);
+    })
+  }
+
+  showRacingResult() {
+    // name : --- 형식으로 출력
+    this.#cars.forEach( car => {
+      Console.print(car.getName() + ' : ' + '-'.repeat(car.getForwardCount()));
+    })
+    Console.print('');
+  }
+
+  Winner() {
+    let max = 0;
+    this.#cars.forEach(car => {
+      let forwardCount = car.getForwardCount();
+      if (max < forwardCount) {
+        max = forwardCount;
+      }
+    })
```

Comment:
```suggestion
const max = Math.max(...this.#cars.map(car => car.getForwardCount()));
```
로 짧게 줄일 수 있을 것 같습니다!